pkg/platform/runtime/buildscript/file.go

```diff
@@ -13,32 +13,37 @@ import (
 	"github.com/ActiveState/cli/pkg/platform/authentication"
 	"github.com/ActiveState/cli/pkg/platform/model"
 	"github.com/ActiveState/cli/pkg/platform/runtime/buildexpression"
-	"github.com/ActiveState/cli/pkg/projectfile"
+	"github.com/ActiveState/cli/pkg/project"
 )
 
-func NewScriptFromProjectDir(dir string, auth *authentication.Auth) (*Script, error) {
-	return newScriptFromFile(filepath.Join(dir, constants.BuildScriptFileName), auth)
+type targeter interface { // note: cannot import runtime/setup.Targeter due to import cycle
+	ProjectDir() string
```

Comment:
If you add an alias for `Dir()` called `ProjectDir()` to the `Project` package then this interface can satisfy both the runtime target instance as well as the project instance, allowing you to consolidate the constructors and remove the direct dependency on the project package.